solr/solrj/src/java/org/apache/solr/common/cloud/ClusterState.java

```diff
@@ -181,29 +183,46 @@ public String getShardId(String nodeName, String coreName) {
   }
 
   public String getShardId(String collectionName, String nodeName, String coreName) {
```

Comment:
Nobody calls this; lets remove it

Reply:
It's a public SolrJ API so I hesitate to remove it in a minor release.

Reply:
Sorry; I defintiely don't think we should hold ourselves to that high of a standard.  As you know there are a ton of classes and methods that are public in all of SolrJ JARs.  The risk of *not* giving ourselves permission to remove stuff is that contributions that might come, instead do not because it's too much work to maintain backwards compatibility.

Reply:
Can mark deprecated for now; don't need to remove it yet.